### PR TITLE
Blog - A Guide to Setting Up Your Monorepo for JavaScript Projects with Lerna

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
Preview URL:
https://63fcd107350dd5007c6ac3a4--ephemeral-narwhal-3ac3b2.netlify.app

File: blog/_posts/2023-04-03-javascript-monorepo-setup-with-larna.md [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+    RUN npx lerna run test
+~~~
+
+The syntax is inspired by Dockerfiles, so it will be familiar if you have worked with Docker before. To test your config, run `earthly +build` and `earthly +test` to run the `build` and `test` steps, respectively. This will download the necessary Docker image and use it to run the build or test scripts of your packages. If everything works, you should see an output that looks like this:
+
+<div class="wide">
+![Earthly success]({{site.images}}{{page.slug}}/q651Aty.png)\
+</div>
+
+If both commands are working locally, they should work in CI as well. To configure a [Github](/blog/ci-comparison Actions to use these, run the following commands to create a new workflow:
```
Comment (balapriyac) [inline blog/_posts/2023-04-03-javascript-monorepo-setup-with-larna.md:421]:
```suggestion
If both commands are working locally, they should work in CI as well. To configure a [Github Actions](/blog/ci-comparison) to use these, run the following commands to create a new workflow:
```

File: blog/_posts/2023-04-03-javascript-monorepo-setup-with-larna.md [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+
+Thankfully, due to its age and popularity, there is a lot of community support for Lerna, so if you have an issue, you'll likely be able to find a solution in one form or another.
+
+It's also worth noting that Lerna isn't the only player in this space anymore. The major package managers (npm, Yarn, and [pnpm](https://pnpm.io/)) now offer workspace features, which can handle a lot of the same use cases as Lerna. Lerna will even leverage these features itself, depending on which package manager you use.
+
+Whether you use Lerna or one of the package managers' solutions, such tools are indispensable when working with monorepos.
+
+## Wrapping Up
```
Comment (balapriyac) [inline blog/_posts/2023-04-03-javascript-monorepo-setup-with-larna.md:488]:
```suggestion
## Conclusion
```

Comment (balapriyac) [reply blog/_posts/2023-04-03-javascript-monorepo-setup-with-larna.md:488]: For consistency, we'll use _Conclusion_ always.

**Review by balapriyac — CHANGES_REQUESTED** [review]
@msali123 
Thanks Sameer. 
- Just left a couple small notes.
- Also, I think Ahmad misspelled **Lerna** in the slug to use in the doc. It's currently `javascript-monorepo-setup-with-larna`. Please update the slug to `javascript-monorepo-with-lerna`.

**Comment by msali123** [comment]
https://6401b76238e2612bf733b540--ephemeral-narwhal-3ac3b2.netlify.app/blog/javascript-monorepo-with-lerna/

@balapriyac done

**Comment by balapriyac** [comment]
> https://6401b76238e2612bf733b540--ephemeral-narwhal-3ac3b2.netlify.app/blog/javascript-monorepo-with-lerna/
> 
> @balapriyac done

Thanks, Sameer. @jalletto this is ready to merge!

**Review by jalletto — APPROVED** [review]

**Pull request merged**